index.html

```diff
@@ -0,0 +1,322 @@
+			</header>
+
+			<div class="list">
+				Size: &nbsp;&nbsp;&nbsp;&nbsp;<span class="strong">17.2 h</span>
+			</div>
+
+			<div class="list">
+				Weight: &nbsp;&nbsp;&nbsp;&nbsp;<span class="strong">602 kg</span>
+			</div>
+
+			<footer class="specs_foot">
+				Specifications valid until 04/08/17
+			</footer>
+
+		</section>									<!-- End <section> .specs -->
+
+		<section class-"pricing">		<!-- Begin <section> .pricing -->
+
+			<header class="h2_title">
+				<h2>Pricing</h2>
+			</header>
+
+			<div class="list">
+				1: &nbsp;&nbsp;&nbsp;&nbsp;<span class="strong">$1,250</span>
+			</div>
+
+			<div class="list">
+				2-5: &nbsp;&nbsp;&nbsp;&nbsp;<span class="strong">$1,200</span>
+			</div>
+
+			<div class="list">
+				5+: &nbsp;&nbsp;&nbsp;&nbsp;<span class="strong">$1,025</span>
+			</div>
+
+		</section>									<!-- End <section> .pricing -->
+
+	</article>
+	<!-- End article 3 -->
+
+
+	<!-- Start article 4 -->
+	<article>
+		<header class="card_title">
+			<h1>The Dawnracer</h1>
+		</header>
+
+		<section class="product">		<!-- Begin <section> .product -->
+
+			<figure class="product_img">
+				<img class="card_img" src="img/unicorn-4.jpg">
+				<figcaption class="fig_hover">
+					Show the sun who's boss
+					<p>
+						Image source: www.fanpop.com
+					</p>
+				</figcaption>
+			</figure>
+
+			<div class="product_desc">
+				<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec vitae sem vel nisi pellentesque semper. Nulla et elit lorem. Vivamus sodales justo mauris, vitae luctus purus condimentum volutpat. Aliquam tempor non lectus a commodo. Nulla nec ultricies nibh, sed interdum metus. Quisque ac tincidunt felis. Sed id sodales nisl.</p>
+			</div>
+
+			<div class="product_avail_logo">
+				Available!
+			</div>
+
+		</section>									<!-- End <section> .product -->
+
+		<section class="specs">			<!-- Begin <section> .specs -->
+
+			<header class="h2_title">
+				<h2>Specifications</h2>
+			</header>
+
+			<div class="list">
+				Size: &nbsp;&nbsp;&nbsp;&nbsp;<span class="strong">16.5 h</span>
+			</div>
+
+			<div class="list">
+				Weight: &nbsp;&nbsp;&nbsp;&nbsp;<span class="strong">554 kg</span>
+			</div>
+
+			<footer class="specs_foot">
+				Specifications valid until 04/08/17
+			</footer>
+
+		</section>									<!-- End <section> .specs -->
+
+		<section class-"pricing">		<!-- Begin <section> .pricing -->
+
+			<header class="h2_title">
+				<h2>Pricing</h2>
+			</header>
+
+			<div class="list">
+				1: &nbsp;&nbsp;&nbsp;&nbsp;<span class="strong">$1,400</span>
+			</div>
+
+			<div class="list">
+				2-5: &nbsp;&nbsp;&nbsp;&nbsp;<span class="strong">$1,350</span>
+			</div>
+
+			<div class="list">
+				5+: &nbsp;&nbsp;&nbsp;&nbsp;<span class="strong">$1,275</span>
+			</div>
+
+		</section>									<!-- End <section> .pricing -->
+
+	</article>
+	<!-- End article 4 -->
+
+</div>
+
+<footer id="main-footer">
+&copy;2016 YOUnicorn Inc.
+</footer>
+
+</body>
+</html>
```

Comment:
Always include a new line at the end of every file. Consider adding ensure_newline_at_eof_on_save to your sublime preferences: http://docs.sublimetext.info/en/latest/reference/settings.html#automatic-behavior